packages/pg/package.json

```diff
@@ -20,6 +20,7 @@
   "author": "Brian Carlson <brian.m.carlson@gmail.com>",
   "main": "./lib",
   "dependencies": {
+    "@peculiar/x509": "^1.12.3",
```

Comment:
should absolutely pin this dependency - i only allow packages within this repo to float patch versions w/ semver. Anything external needs to be vetted & checked on exact versions, and using exact versions in package.json please!

Reply:
Sure.

Reply:
See also comments below.